src/routing/core.js

```diff
@@ -33,7 +33,8 @@ const coreBuildPaths = [
   "/dengue",
   "/ebola",
   "/enterovirus",
-  "/flu",
+  "/seasonal-flu",
+  "/avian-flu",
   "/lassa",
   "/measles",
```

Comment:
Suggestion: add new entries in alphabetical order for consistency

Reply:
Done - thanks!